modules/administration-guide/pages/configuring-editors-definitions.adoc

```diff
@@ -174,6 +174,19 @@ commands:
 
 . Refresh the {prod-short} Dashboard page to see new available editor.
 
+== Retrieving the editor definition
+
+The editor definition is also served by the {prod-short} dashboard API from the following URL:
+
+`pass:c,a,q[{prod-url}]/dashboard/api/editors/devfile?che-editor=__<editor id>__`
+
+For the example from xref:configuring-editors-definitions.adoc[], the editor definition can be retrieved by accessing the following URL:
+
+`pass:c,a,q[{prod-url}]/dashboard/api/editors/devfile?che-editor=publisher/editorname/version`
+
+TIP: When retrieving the editor definition from within the {orch-name} cluster, the {prod-short} dashboard API can be accessed via the dashboard service:
+`\http://che-dashboard.{prod-namespace}.svc.cluster.local:8080/dashboard/api/editors/devfile?che-editor=__<editor id>__`
```

Comment:
I am getting this error:
```
Get "http://che-dashboard.eclipse-che.svc.cluster.local:8080/dashboard/api/editors/devfile": dial tcp: lookup che-dashboard.eclipse-che.svc.cluster.local: no such host --- docs/next/end-user-guide/managing-workspaces-with-apis/index.html --> http://che-dashboard.eclipse-che.svc.cluster.local:8080/dashboard/api/editors/devfile?che-editor=che-incubator/che-code/latest
```

Hi @deerskindoll is there a way to supress the error? This link is not supposed to be accessible unless it's accessed within a cluster

Reply:
on it

Reply:
@dkwon17 you need to add `pass:c,a,q` in front of the address to deactivate the link

additional thought: since the address is explicitly featuring che-dashboard, is it applicable for downstream? otherwise, you need to replace it with {prod-url}

Reply:
Updated, it is working, thank you

Good catch, it should be `che` upstream and `devspaces` downstream, so I've updated it to `{prod-id-short}`